bundle.yml

```diff
@@ -66,6 +77,8 @@ options :
             ENABLE_CLOUDSC_GPU_SCC_CUF=ON
             ENABLE_CLOUDSC_GPU_SCC_CUF_K_CACHING=ON
             ENABLE_CLOUDSC_GPU_SCC_FIELD=ON
+            ENABLE_FIELD_API=ON
```

Comment:
Remove `ENABLE_FIELD_API=ON` and `ENABLE_CLOUDSC_GPU_SCC_FIELD=ON` (see comment in CMakeLists.txt) but retain `BUILD_field_api=ON`